scripts/run_tests.bat

```diff
@@ -0,0 +1,27 @@
+@echo off
+
+echo.
+echo Creating virtual environment
+echo.
+pip3.exe install virtualenv --user
+python3.exe -m venv ./.virtualenv
+
+echo.
+echo Running virtual environment
+echo.
+call .\.virtualenv\Scripts\activate.bat py35
+
+pushd condensation-forum
+
+echo Installing prerequisites
+echo.
+pip3 install -r requirements.txt
+echo.
+echo Running application
+echo.
+python3 _test_schema.py
```

Comment:
Quick note: the calls above are made `pip3.exe` and `python3.exe` because some shells do not resolve partial names for applications. Example: Git Bash/MinTTY. Let's just include `.exe` on Windows executables.